homeassistant/components/meteo_france/const.py

```diff
@@ -24,96 +28,96 @@
 ENTITY_NAME = "name"
 ENTITY_UNIT = "unit"
 ENTITY_ICON = "icon"
-ENTITY_CLASS = "device_class"
+ENTITY_DEVICE_CLASS = "device_class"
 ENTITY_ENABLE = "enable"
 ENTITY_API_DATA_PATH = "data_path"
 
 SENSOR_TYPES = {
     "pressure": {
         ENTITY_NAME: "Pressure",
         ENTITY_UNIT: PRESSURE_HPA,
-        ENTITY_ICON: "mdi:gauge",
-        ENTITY_CLASS: "pressure",
+        ENTITY_ICON: None,
+        ENTITY_DEVICE_CLASS: DEVICE_CLASS_PRESSURE,
         ENTITY_ENABLE: False,
         ENTITY_API_DATA_PATH: "current_forecast:sea_level",
     },
     "rain_chance": {
         ENTITY_NAME: "Rain chance",
         ENTITY_UNIT: UNIT_PERCENTAGE,
         ENTITY_ICON: "mdi:weather-rainy",
-        ENTITY_CLASS: None,
+        ENTITY_DEVICE_CLASS: None,
         ENTITY_ENABLE: True,
         ENTITY_API_DATA_PATH: "probability_forecast:rain:3h",
     },
     "snow_chance": {
         ENTITY_NAME: "Snow chance",
         ENTITY_UNIT: UNIT_PERCENTAGE,
         ENTITY_ICON: "mdi:weather-snowy",
-        ENTITY_CLASS: None,
+        ENTITY_DEVICE_CLASS: None,
         ENTITY_ENABLE: True,
         ENTITY_API_DATA_PATH: "probability_forecast:snow:3h",
     },
     "freeze_chance": {
         ENTITY_NAME: "Freeze chance",
         ENTITY_UNIT: UNIT_PERCENTAGE,
         ENTITY_ICON: "mdi:snowflake",
-        ENTITY_CLASS: None,
+        ENTITY_DEVICE_CLASS: None,
         ENTITY_ENABLE: True,
         ENTITY_API_DATA_PATH: "probability_forecast:freezing",
     },
     "wind_speed": {
         ENTITY_NAME: "Wind speed",
         ENTITY_UNIT: SPEED_KILOMETERS_PER_HOUR,
         ENTITY_ICON: "mdi:weather-windy",
-        ENTITY_CLASS: None,
+        ENTITY_DEVICE_CLASS: None,
         ENTITY_ENABLE: False,
         ENTITY_API_DATA_PATH: "current_forecast:wind:speed",
     },
     "next_rain": {
         ENTITY_NAME: "Next rain",
         ENTITY_UNIT: None,
         ENTITY_ICON: "mdi:weather-pouring",
```

Comment:
Please remove the custom icon.

Reply:
I've removed it first but I saw there is no icon for this device_class, it shown the generic icon of an entity (the eye).

Is no icon for the `timestamp` device_class is a bug ?
If not, can I keep this custom icon to replace the generic icon ?

Reply:
It should be added in the frontend in that case. We don't set custom icons for entities with device class set.

If you're using the lovelace entities card it will show an appropriate icon at least.

![timestamp_sensor_entities](https://user-images.githubusercontent.com/3181692/90019195-611e0900-dcae-11ea-9771-02297c70060f.png)

Reply:
This is not that I got ...

<img width="402" alt="Capture d’écran 2020-08-12 à 23 18 02" src="https://user-images.githubusercontent.com/14821482/90069145-2a69e200-dcf2-11ea-83e2-2f3b6893747c.png">